doc/cephfs/quota.rst

```diff
@@ -23,9 +23,12 @@ Limitations
    of data.  Generally speaking writers will be stopped within 10s of
    seconds of crossing the configured limit.
 
-#. *Quotas are not yet implemented in the kernel client.* Quotas are
-   supported by the userspace client (libcephfs, ceph-fuse) but are
-   not yet implemented in the Linux kernel client.
+#. *Quotas are implemented in the kernel client > 4.16 only.* Quotas
+   are supported by the userspace client (libcephfs, ceph-fuse).
+   Linux kernel clients > 4.16 support CephFS quotas but only on
```

Comment:
In fact, initially I had  ">= 4.17" and changed it to "> 4.16" because of the rumors regarding a 5.0 :-)
But yeah we can fix that later if that's the case.